bin/systemd_module_migration.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Do we really want to use bash(1)?  If so:
```suggestion
#!/usr/bin/env bash
```

otherwise a fine:
```suggestion
#!/bin/sh
```

Most *BSD systems do not ship with bash(1) and do not install it in `/bin`, so I trend to always stick to using `/bin/sh` and avoid non-POSIX-compliant code :wink: